src/common/theme/colors.ts

```diff
@@ -56,7 +56,7 @@ export const colors = {
     800: '#991B1B',
     900: '#7F1D1D',
   },
-  blue: {
+  ['blue-secondary']: {
```

Comment:
@tzeweiwee renamed this to `blue-secondary` because I saw 2 blue property in the object (colors)

Reply:
ah that's why, i think these 'secondary' colors are not needed (may have added in the beginning but not removed), `blue.200: #B6E8F7` is used. Could you help to remove them?

Reply:
i think blue.200 is #E9F9FF